hack/go-vet.sh

```diff
@@ -14,19 +14,16 @@
 # limitations under the License.
 
 
-if [[ -z ${CI} ]]; then
-    ./hack/go-mod.sh
```

Comment:
Because of this condition the script was not executing. In circle CI this is an enviornment variable set to `true`

Reply:
You want this to be executed to catch when someone send a PR without running `make mod`?

Reply:
Not just this step, but all the steps were in this condition like operator-sdk generate k8s and crds, which were not executed on this CircleCI

Reply:
Because of this condition the script `go-vet.sh` was only running a single command `go vet ./...`